### PR TITLE
[FIX] mail: fix line break in customer rating feedback

### PR DESCRIPTION
**Pull request by dhrs-odoo**
In this PR we have fixed the line break issue when we submit a feedback for task and ticket .

steps:
----
   - Install helpdesk
   - Go to helpdesk > team
   - Activated customer rating
   - Set the rating template in the ticket stage
   - Create a ticket with the customer and move to the next  stage
   - Open the mail and submit a rating and message
   - Open ticket and check customer message
  
task-3268763

### CODE REVIEW

**Review by auon-odoo — APPROVED** [review]

File: addons/mail/static/src/core/common/message.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
                             </div>
                         </div>
                         <div
-                            class="position-relative d-flex"
+                            class="position-relative align-items-center"
```
Comment (xavierbol) [inline addons/mail/static/src/core/common/message.xml:67]: TBH, I don't understand why this changes fixes the issue. 🤯

Comment (xavierbol) [reply addons/mail/static/src/core/common/message.xml:67]: @dhrs-odoo ?

**Comment by dhrs-odoo** [comment]
Hello @xavierbol 
Sorry for the delay , so the previous fix was not so accurate and I checked it and somehow it was not working but now I have added the updated fix.
Thank You :innocent:

**Comment by xavierbol** [comment]
@dhrs-odoo the bug is only in master and so due to a recent changes in mail? 🧐

**Comment by dhrs-odoo** [comment]
Hello @xavierbol The bug is since saas-16.3 and at the time of fix it was master 
So need to retarget the branch to saas-16.3 :innocent:

**Comment by xavierbol** [comment]
Hello @dhrs-odoo 

Can you retarget your fix in 16.3?

Thanks in advance!

**Comment by ppr-odoo** [comment]
> Hello @dhrs-odoo
> 
> Can you retarget your fix in 16.3?
> 
> Thanks in advance!

done.thanks

**Comment by xavierbol** [comment]
Thanks again for your work. 🙂

**Comment by xavierbol** [comment]
Hello @odoo/rd-discuss ,

Can you review that fix when you have time? 🙂 

TBH, I am not sure if it is the right fix. 🤔 

Thanks in advance!

Have a nice day! 🙂

**Comment by seb-odoo** [comment]
Hello, there is no explanation anywhere about what we are trying to fix here?
How to reproduce, what is the result before/after?

**Comment by ppr-odoo** [comment]
> Hello, there is no explanation anywhere about what we are trying to fix here? How to reproduce, what is the result before/after?

The commit message and PR description now include a step that I added. I request that you take a look at the image below.

image

**Review by alexkuhn — APPROVED** [review]
LGTM. Could probably use `d-inline-block` instead to match with original `o-mail-Message-body`.

Duplicated code in original code and patch here is accidental. Using same class would ease code cleaning in master. Thanks!

**Comment by tong-odoo** [comment]
Hello @dhrs-odoo @xavierbol, may I ask why you will apply d-inline-flex to the message.body? If apply d-inline-flex, you are assuming the message doesn't have many children doms. It works on customer rating feedback but may not work on other message. 
i.e. the message after filling in the salary configurator
image

Could we update the customer rating message body instead? Like add a wrapper on the image and message and add the class inside the wrapper? https://github.com/odoo/odoo/blob/17.0/addons/rating/models/mail_thread.py#L150

**Comment by seb-odoo** [comment]
Hello, the change is being reverted here: https://github.com/odoo/odoo/pull/149516

**Comment by xavierbol** [comment]
Hello @tong-odoo,

It was to fix the issue mentioned in the task linked and explained in that PR (to correctly render the customer rating feedback message), we did not know it will break other templates. Also, as I said in one of my previous comments, I did not know if it was the right fix, that's why I ask a review to Discuss team and we did not see it could break other templates. So, indeed, it was wrong, sorry about that. 

If you have a better solution to fix our issue then feel free to create a PR. Otherwise, we will again check how to fix that issue in other way. 🙂

Sorry again!

Have a nice day! 🙂 

cc @dhrs-odoo

**Comment by xavierbol** [comment]
> Hello, the change is being reverted here: #149516

Thanks for the quick fix. 🙂
